.rubocop.yml

```diff
@@ -0,0 +1,6 @@
+Documentation:
+  Enabled: false
+
+LineLength:
+  Max: 99
+  
```

Comment:
You can add:
```
Style/FrozenStringLiteralComment:
  Enabled: false
```
To avoid the `# frozen_string_literal: true`. 
So, after introducing this piece of code, remove all `# frozen_string_literal: true` instances.